.github/workflows/release.yml

```diff
@@ -0,0 +1,37 @@
+on:
+  push:
+    # Sequence of patterns matched against refs/tags
+    tags:
+    - 'v*' # Push events to matching v*, i.e. v1.0, v20.15.10
+
+name: Upload Release Asset
+
+jobs:
+  build:
+    name: Upload Release Asset
+    runs-on: macOS-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+      - name: Build project
+        run: make archive 
+      - name: Create Release
+        id: create_release
```

Comment:
I'm not sure the `id` field is needed here or below.

Reply:
Er, scratch that. I see why it's needed here but might not be needed below.

Reply:
This workflow is based on these examples from GH

https://github.com/actions/create-release
https://github.com/actions/upload-release-asset